guides/installing_esphome.rst

```diff
@@ -0,0 +1,97 @@
+Installing ESPHome
+==================
+
+Windows
+-------
+
+Download Python from `the official site <https://www.python.org/downloads/>`_.
+
+.. figure:: images/python-win-installer.png
+    :align: center
+    :width: 75.0%
+    :alt: Python installer window with arrows pointing to "Add Python to PATH" and "Install Now"
```

Comment:
For some reason this image is clickable.

Reply:
All images are clickable, as far as I can see. If you don't like this, it can probably be changed in the configuration, but it's not something I touched in this PR.